### DISCUSSION TITLE
GPU and JIT talk

### DISCUSSION THREAD

**Discussion by Padarn** [pyg-team/pytorch_geometric#2286] [Q&A] [answered]
@rusty1s and @lgray I just watched your talk here: https://www.youtube.com/watch?v=4swsvOLzL_A, thanks a lot it was really interesting.

Sorry I have a basic question: It is mentioned that the JIT allows you to not depend on CUDA libraries. Can you explain why this is?

**Comment by rusty1s** [comment, marked as answer]
The JIT allows us to be independent of a Python environment, while CUDA is still required in case you want to execute your GNN on a GPU.

**Reply by Padarn** [reply]
Got it. Must have misunderstood that from the talk. Thanks
